### ISSUE TITLE
Counter issue

### ISSUE THREAD

**Issue by deepaksaipendyala** [antonkomarev/github-profile-views-counter#43]
The views counter raising rapidly!

**Comment by deepaksaipendyala** [comment]
It is around 1k few days back but now it is 5k 6k 7k....

**Comment by antonkomarev** [comment]
Usually this may happen when you have some kind of installed browser plugins which visit your profile page

**Comment by japzio** [comment]
is it possible to not count your own view? it seems like it is also counted if i view my own profile.

**Comment by antonkomarev** [comment]
@japzio no, you can read details here #11 and [here](https://github.com/antonkomarev/github-profile-views-counter#why-does-the-counter-increase-every-time-the-page-is-reloaded)

**Comment by japzio** [comment]
@antonkomarev ok got it. Thanks
